core/include/clist.h

```diff
@@ -372,8 +372,8 @@ typedef int (*clist_cmp_func_t)(clist_node_t *a, clist_node_t *b);
  *
  * @internal
  *
- * @param[in]   list    ptr to first element of list
- * @param[in]   cmp     comparison function
+ * @param[in]   list_head   ptr to the clist
```

Comment:
I think the "ptr to first element of list" is accurate

Reply:
No, it is a ptr to an element that is no part of the circular list, one can either name it head or list but its is not the first elment

Reply:
What you say is accurate for `clist_sort()`, but this (`_clist_sort()`) is the helper that gets called by `clist_sort()`with the actual, first element (list head, or "l").

Reply:
I am sorry `_clist_sort` does not work on clist but on the list inside. - `clist_sort` unpacks the clist to the list

you are right this function is not as the others.

Reply:
i fixed that change back 

(somehow github does no longer update comments while in stay on a PR, maybe some new javascript url need to be allowed)

Reply:
for some reason this still shows up as a change request but since the clist part got its own PR(s) (#16311 and #16330) this is no longer part of this PR